pyproject.toml

```diff
@@ -0,0 +1,16 @@
+[tool.poetry]
+name = "my-kmeans"
+version = "0.1.0"
```

Comment:
My version is 999.999.999. Please change

Reply:
(T T)

Reply:
changed as you said!
[26d5118](https://github.com/matsui528/my_kmeans/pull/4/commits/26d51187a8735450de8e84dae521c714f41a8214)